app.js

```diff
@@ -129,29 +188,59 @@ function header(message) {
  * @param {puppeteer.Browser} browser
  * @returns {Promise<boolean>} Are there any drops?
  */
-async function hasValorantDrop(browser) {
+async function hasDrops(browser) {
```

Comment:
I'd prefer if we replaced the nested-ifs with guard-style ifs

Reply:
Ok, can you do that, I don't really know where you're refering?